plugins/NavStars/src/gui/NavStarsWindow.cpp

```diff
@@ -251,6 +251,7 @@ void NavStarsWindow::populateNavigationalStarsSets()
 	nsSets->addItem("Celestron", "Celestron");
 	// TRANSLATORS: Part of full phrase: Apollo space program set of navigational stars
 	nsSets->addItem(q_("Apollo space program"), "Apollo");
+	nsSets->addItem("Skywatcher", "Skywatcher");
```

Comment:
I think the Apollo list should go between the Nautical lists and the telescope lists.